### DISCUSSION TITLE
How to correctly export a model for ONNX Runtime Web

### DISCUSSION THREAD

**Discussion by meded90** [microsoft/onnxruntime#21613] [Web Q&A]
I use the standard export.
```python
  model = YOLO(latest_run_folder + '/weights/best.pt')
  onnx_model_path = latest_run_folder + '/weights/best.onnx'
  model.export( format='onnx',dynamic=True)
```
but I get errors like this
![image](https://github.com/user-attachments/assets/f72136b0-04c1-42d5-94b5-267dfeb53077)

**Comment by gyagp** [comment]
Current ONNX Runtime WebGPU impl has a constraint that axis of Softmax must be the last. We need to loose this constraint.
